lib/danger_plugin.rb

```diff
@@ -15,7 +15,9 @@ module Danger
   #      scheme: 'EasyPeasy',
   #      workspace: 'Example/EasyPeasy.xcworkspace',
   #      exclude_targets: 'Demo.app',
-  #      minimum_coverage_percentage: 90
+  #      minimum_coverage_percentage: 90,
+  #      minimum_coverage_percentage_for_changed_files: 80.0,
+  #      filename_prefix_ignore_list: ['View', 'Cell', 'Layout', 'Action', 'State'],
```

Comment:
the names are getting pretty long already, but I think we should express that the ignore list only applies to the changed files, so maybe `minimum_coverage_percentage_for_changed_files_ignore_list` would be a better name 🤔

Reply:
Yeah, I agree that making the intent clear is more sensible even if it is with long names. I would like to go with `ignore_list_of_minimum_coverage_percentage_for_changed_files`, let me know if that makes sense or we can change it.